node/src/components/rpc_server/rpcs/docs.rs

```diff
@@ -37,11 +37,11 @@ const DEFINITIONS_PATH: &str = "#/components/schemas/";
 // As per https://spec.open-rpc.org/#service-discovery-method.
 pub(crate) static OPEN_RPC_SCHEMA: Lazy<OpenRpcSchema> = Lazy::new(|| {
     let contact = OpenRpcContactField {
-        name: "CasperLabs".to_string(),
+        name: "Casper Labs".to_string(),
         url: "https://casperlabs.io".to_string(),
     };
     let license = OpenRpcLicenseField {
-        name: "CasperLabs Open Source License Version 1.0".to_string(),
+        name: "Casper Open Source License Version 1.0".to_string(),
```

Comment:
I think this needs to be Apache 2.0 - not COSL.

Reply:
@MParlikar Updated to the Apache license.